find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
```

Comment:
max_index as a variable name sounds misleading, is this variable supposed to represent  the length of the array?